src/schema/objects/metadata.yaml

```diff
@@ -466,6 +466,31 @@ Columns:
   type: array
   items:
     type: string
+ColumnDescription:
```

Comment:
How do people feel about this being in `objects.metadata`? This describes an object in a sidecar to a TSV file. For example, if my TSV has a column `handedness`, my JSON could have:

```JSON
{
  "handedness": {
    "Description": "Edinburgh handedness inventory",
  }
}
```

This introduces `name: '.*'` to indicate that the column name could be anything. But perhaps we want something else, like `objects.structures` or `objects.types`.